src/editor/components/SearchResults.js

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+import { TouchableOpacity, Text, StyleSheet } from 'react-native';
+
+const SearchResults = ({
+  name,
+  id,
+  setPreviewInfo,
+  setHighlightedID,
+  isHighlighted,
+}) => {
+  const display_preview = () => {
+    const baseQuery = 'https://www.googleapis.com/drive/v3/files/';
```

Comment:
you can use DRIVE_API_URLS.SEARCH_FILES (from config.json in the services folder) here instead, for consistency